### ISSUE TITLE
Cluster bootstrap script collection

### ISSUE THREAD

**Issue by ARMmaster17** [ARMmaster17/Captain#4]
Since in theory it would be possible to bootstrap an entire Captain cluster with only a builder node, it would be nice to have an AIO script to build that bootstrap node (and possibly delete when done?) Two options:
1. Build a container with etcd, RabbitMQ, and builder all on the same node. That builder then builds a new etcd, RabbitMQ, builder, and the rest of the cluster (and then imports the existing containers into flightplan).
2. E/R/B containers are set up manually (through AIO script) and then builder forms part of the new cluster that it then builds.

**Comment by ARMmaster17** [comment]
Current `--bootstrap` flag will create an ATC node and a PostgreSQL database. Installing ATC and configuring Postgres is currently blocked by #14.
